### PR TITLE
banner timings change plus 2 added banner messages

### PR DESCRIPTION
**Pull request by colin-hillis**
- Banner timing reduced from 8 to 5 seconds
- 2 new banner messages added. One announcing Beta 2.0 release and one signposting the recently archived docs.

### CODE REVIEW

File: themes/doks/layouts/partials/header/header.html [hunk 1]
```diff
@@ -4,11 +4,17 @@
...
     <!-- Uncomment, edit and duplicate as necassary -->
-    <!-- <p class="banner-message__item" data-banner-message>Message two...</p> -->
-  </div>
+    <p class="banner-message__item" data-banner-message>
+        Recently archived versions of 'Corda Enterprise' and 'Corda Open Source' documentation can be found <b><a style ="color:#FFFFFF; text-decoration: underline;" href="https://github.com/corda/corda-docs-portal/tree/main/content/en/archived-docs">HERE.</a></b>
+    </p>
+        <!-- Uncomment, edit and duplicate as necassary -->
+        <p class="banner-message__item" data-banner-message>
+          Corda 5 Beta 2.0 is coming soon. Release is planned for 22nd March 2023.
+      </p>
```
Comment (yarivus) [inline themes/doks/layouts/partials/header/header.html:16]: ensure the right indentation here

**Review by yarivus — CHANGES_REQUESTED** [review]

**Comment by colin-hillis** [comment]
Here is the preview for doing review: https://colin.preview.docs.r3.com/

**Review by yarivus — COMMENTED** [review]
@colin-hillis 
image
There's still a gap between the red banner and the dark grey stretch

**Comment by colin-hillis** [comment]
> 

That has nothing to do with this PR though. It is with FE team as it relates to their PR.

**Comment by yarivus** [comment]
> > 
> 
> That has nothing to do with this PR though. It is with FE team as it relates to their PR.

So there's an active PR to fix it ? 
@conor-r3 for awareness

**Comment by colin-hillis** [comment]
> 

Conor and I spoke about it yesterday and Peter is on it.

**Review by paulmoloneyr3 — DISMISSED** [review]

**Comment by colin-hillis** [comment]
I had a call with Peter and a fix was introduced to address the banner spacing.

**Review by yarivus — APPROVED** [review]
LGTM

**Review by pbienek-r3 — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**